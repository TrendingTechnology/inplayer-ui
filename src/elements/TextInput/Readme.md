Text input is a input component for text. You can use it directly like this:

```jsx static
import { TextInput } from '@inplayer-org/inplayer-ui'

function Page(props) {
  return <TextInput />;
}
```

### Demo

```jsx
<TextInput placeholder="text" />
```
