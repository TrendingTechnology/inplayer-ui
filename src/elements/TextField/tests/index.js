import { generateSnapshots } from 'helpers';

import TextInput from '../index';

generateSnapshots(TextInput);
