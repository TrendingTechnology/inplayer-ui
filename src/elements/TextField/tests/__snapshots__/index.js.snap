// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField renders correctly 1`] = `
.c0 {
  vertical-align: middle;
  box-sizing: border-box;
  overflow: hidden;
  outline: none;
  color: #44494e;
  -webkit-transition: all ease 300ms;
  transition: all ease 300ms;
  background: #fff;
  font-weight: 300;
  border: 1px solid #d7dde5;
  padding: 10px 18px 11px;
  border-radius: 3px;
  font-size: 1rem;
  line-height: 1;
  min-width: 260px;
  max-width: 560px;
  width: 100%;
  margin: 6px 0;
}

.c0:focus {
  border-bottom: 1px solid #07AAE6;
}

<input
  className="c0"
/>
`;
